src/scss/components/_listings.scss

```diff
@@ -50,21 +80,149 @@ $listing-borderColor: $color-border;
+            width: 100%;
+            height: auto;
+        }
+
         .c-listing-item-info {
+            top: -4px;
+            position: relative;
             color: $color-text;
-            flex-wrap: nowrap;
-            margin-left: spacing(base);
+            padding-left: 66px;
+
+            @include media('>wide') {
+                position: relative;
+                padding-left: 80px;
+                padding-right: 260px;
+
+                &:after {
+                    content: '';
+                    top: 4px;
+                    width: 1px;
+                    right: 259px;
+                    height: 100%;
+                    display: block;
+                    position: absolute;
+                    background: $listing-borderColor;
+
+                    .c-listing--offline & {
+                        background: $listing-offlineBorderColor;
+                    }
+                }
+            }
+        }
+
+        .c-listing-item-title {
+            word-break: break-word;
         }
+
+        .c-listing-item-meta {
+            top: -3px;
+            float: right;
+            position: relative;
+
+            @include media('>wide') {
+                top: auto;
+                width: 260px;
+                padding-bottom: 50px;
+                padding-left: spacing(x2);
+                border-left: 1px solid $listing-borderColor;
+
+                .c-listing--offline & {
+                    border-color: $listing-offlineBorderColor;
+                }
+            }
+        }
+
+        .c-listing-item-rating {
+            padding: 2px 0 3px;
+
+            .c-rating,
+            .c-listing-item-ratingText {
+                vertical-align: middle;
+            }
+        }
+
+        .c-listing-item-details {
+            width: 100%;
+            padding-left: 66px;
+
+            @include media('>wide') {
+                right: 0;
+                width: 260px;
+                margin-top: 10px;
+                position: absolute;
+                bottom: spacing(x2);
+                padding-left: spacing(x2);
+            }
+        }
+
+        .c-listing-item-distance {
+            position: absolute;
+            right: spacing(x2);
+            bottom: spacing(x2);
+
+            @include media('>mid') {
+                right: 0;
+            }
+
+            @include media('>wide') {
+                position: static;
+                line-height: 0.8;
+                padding-left: 80px;
+                margin-bottom: spacing() / 2;
+            }
+        }
+
+        .c-listing-item-distanceIcon {
+            fill: $listing-iconFill;
+        }
+
         .c-listing-item-text {
             margin-top: 0;
         }
 
-        .c-listing-item-col {
-            border-right: 1px solid $listing-borderColor;
-            margin-left: spacing(x2);
-            padding-left: spacing(x2);
+        .c-listing-item-label,
+        .c-listing-item-flag {
```

Comment:
Probably better to name this to do with the content it's showing in the listing item, rather than how it looks as the design could change and it not look like a flag in future.

Maybe `c-listing-item-promo` or something similar.